pages/pokedex/boomsweeper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Boomsweeper from '../../views/Pokedex/Boomsweeper';
+import { useRouter } from 'next/router';
+import { LAYOUT_TYPE } from '../../lib/constants/layout';
+
+export default function BoomsweeperPage() {
+  const router = useRouter();
+
+  const seed = router.query.seed ?? ''
+  let layout = router.query.layout;
+
+  if (
+    layout !== LAYOUT_TYPE.numerical &&
+    layout !== LAYOUT_TYPE.alphabetical &&
+    layout !== LAYOUT_TYPE.random
+  ) {
```

Comment:
Minor suggestion:
`if (!Object.values(LAYOUT_TYPE).includes(layout))`

Reply:
Aha! So there is an easy way to do this!

Reply:
Yeah. The `Object` object has some very useful helper functions built-in. I use `Object.values` and `Object.keys` a lot.